app/policies/portal/permission_form_policy.rb

```diff
@@ -0,0 +1,18 @@
+class Portal::PermissionFormPolicy < ApplicationPolicy
+
+  def index?
+    admin_or_manager_project_admin?
```

Comment:
Actually based on the other method names you could just call it `manager_or_project_admin`. Since other methods like `manager_or_researcher?` implicitly allow 'admin'.